j2735_msgs/msg/ControlValue.msg

```diff
@@ -0,0 +1,50 @@
+#
+# ControlValue.msg
+#
+# Part of the CARMA Cloud geo-fence specification not part of the official SAE j2735 standard
+#
+# J2735 2016 message format.
+#
+# @author Misheel Bayartsengel
+# @version 0.1
+#
+# Description
+# ...
+
+# ControlValue ::= EXPLICIT CHOICE {
+# 	value INTEGER,
+# 	direction ENUMERATED {forward, reverse},
+# 	lataffinity ENUMERATED {left, right},
+# 	perm ENUMERATED {none, permitted, passing_only, emergency_only},
+# 	prkingallowd ENUMERATED {no, parallel, angled}
+# }
+
+# Value ::= INTEGER (SIZE(32))
```

Comment:
Having SIZE(32) here makes it unclear since that is only used for arrays elsewhere. Just remove the size description.